docker/Dockerfile

```diff
@@ -17,5 +17,5 @@ RUN micromamba install -y -n base git -f /tmp/env.yaml && \
 # Ensure that conda environment is automatically activated
 # https://github.com/mamba-org/micromamba-docker#running-commands-in-dockerfile-within-the-conda-environment
 ARG MAMBA_DOCKERFILE_ACTIVATE=1
-RUN python -m pip install --no-deps .
-ENTRYPOINT ["/usr/local/bin/_entrypoint.sh", "fah-alchemy"]
+RUN python -m pip install --no-deps -e .
```

Comment:
Is it generally more desirable to have an editable install? Just curious on motivation here.

Reply:
It doesn't really matter, it isn't my preference but it was needed when I created the container. I can re-visit that. I was having issues with a "normal" install and this fixed it. I think there have been some changes to the package since I first started working on it, so it will be good to check!

Reply:
All good, fine to leave it!